assets/sass/components/_topics-list.scss

```diff
@@ -1,29 +0,0 @@
-.topics-list {
```

Comment:
Tidying up - File removed as I think it's the old topics list from the media pages.